ocf_data_sampler/torch_datasets/xarray_compute.py

```diff
@@ -0,0 +1,8 @@
+def compute(xarray_dict: dict) -> dict:
```

Comment:
I think it could be nice to put this file and the `process_and_combine.py` file in a common folder or utility folder (in the `torch_datasets` folder), would help make it clearer which files contain the torch datasets